README.md

```diff
@@ -920,10 +920,8 @@ git remote -v
 To pull all the data from the remote into the local repository. Use the `fetch` and `merge` subcommands. Before doing this, check that the current branch is set to `master`.
```

Comment:
This should then be changed to omit the fetch/merge commands, to something like:
"To update the remote into the local repository. Use the `pull` subcommand. Before doing this, make certain the current branch is set to `master`."

Reply:
I think this is largely a matter of preference. I tend to use `git fetch` and `git rebase`. If we're going to suggest `git pull`, I suggest adding `--rebase`.